integration-tests/src/test/java/oracle/weblogic/kubernetes/ItTwoDomainsLoadBalancers.java

```diff
@@ -941,25 +945,35 @@ private void createTwoDomainsOnPVUsingWlstAndVerify() {
 
       // create persistent volume and persistent volume claims
       Path pvHostPath = get(PV_ROOT, this.getClass().getSimpleName(), domainUid + "-persistentVolume");
-
-      logger.info("Creating PV directory {0}", pvHostPath);
-      assertDoesNotThrow(() -> deleteDirectory(pvHostPath.toFile()), "deleteDirectory failed with IOException");
-      assertDoesNotThrow(() -> createDirectories(pvHostPath), "createDirectories failed with IOException");
-
+      if (!OKE_CLUSTER) {
+        logger.info("Creating PV directory {0}", pvHostPath);
+        assertDoesNotThrow(() -> deleteDirectory(pvHostPath.toFile()), "deleteDirectory failed with IOException");
+        assertDoesNotThrow(() -> createDirectories(pvHostPath), "createDirectories failed with IOException");
+      }
       V1PersistentVolume v1pv = new V1PersistentVolume()
           .spec(new V1PersistentVolumeSpec()
               .addAccessModesItem("ReadWriteMany")
-              .storageClassName(domainUid + "-weblogic-domain-storage-class")
               .volumeMode("Filesystem")
               .putCapacityItem("storage", Quantity.fromString("2Gi"))
-              .persistentVolumeReclaimPolicy("Retain")
-              .hostPath(new V1HostPathVolumeSource()
-                  .path(pvHostPath.toString())))
+              .persistentVolumeReclaimPolicy("Retain"))
           .metadata(new V1ObjectMetaBuilder()
               .withName(pvName)
               .build()
               .putLabelsItem("weblogic.domainUid", domainUid));
 
+      if (OKE_CLUSTER) {
```

Comment:
Similarly, this code block is also repeated. Perhaps we need a class with utility methods to fix-up the PV and PVC for different environments.

Reply:
consolidated as util method